basics/Main.java

```diff
@@ -0,0 +1,72 @@
+import java.time.LocalDateTime;
+
+public class Main {
+    
+public static void main(String[] args) {
+int dogCount = 1;
+System.out.println("I own " + dogCount + " " + pluralize("dog", dogCount) + ".");
+
+int catCount = 2;
+System.out.println("I own " + catCount + " " + pluralize("cat", catCount) + ".");
+
+int turtleCount = 0;
+System.out.println("I own " + turtleCount + " " + pluralize("turtle", turtleCount) + ".");
+
+flipNHeads(1);
+
+Main object = new Main();
+object.clock();
+}
+
+
+
+public static String pluralize(String word, int number) {
+
+    if (number == 1) {
+    return word;
+    } else {
+    return word + "s";
+}
+}
+
+public static void flipNHeads(int n) {
+
+int counter = 0;
+int headCounter = 0;
+
+while (headCounter<n) {
+    counter++;
+if (Math.random() < 0.5){
+System.out.println("Tails");
+} else {
+System.out.println("Heads");
+headCounter ++;
+
+}
+}
+System.out.println("it took " + counter + " flips to flip " + n + " heads in a row");
+}
+
+
+//Stackoverflow and Crunchify as resource
```

Comment:
I'd love if you included the specific links you used at this point in your file. This can also help your future self, when you come back to look at how you approached this problem, and you have links to the resources you used!